tasks.py

```diff
@@ -20,20 +32,24 @@ def is_truthy(arg):
     return bool(strtobool(arg))
 
 
-COMPOSE_FILES = ["docker-compose.yml", "../docker-compose.override.yml"]
-if os.getenv("NAUTOBOT_USE_MYSQL"):
-    COMPOSE_FILES.append("docker-compose.mysql.yml")
-
+# Use pyinvoke configuration for default values, see http://docs.pyinvoke.org/en/stable/concepts/configuration.html
+# Variables may be overwritten in invoke.yml or by the environment variables INVOKE_NAUTOBOT_GOLDEN_CONFIG_xxx
 namespace = Collection("nautobot_golden_config")
 namespace.configure(
     {
         "nautobot_golden_config": {
             "nautobot_ver": "1.2.1",
             "project_name": "nautobot_golden_config",
-            "python_ver": "3.7",
+            "python_ver": "3.8",
             "local": False,
             "compose_dir": os.path.join(os.path.dirname(__file__), "development"),
-            "compose_files": COMPOSE_FILES,
+            "compose_files": [
+                "docker-compose.base.yml",
+                "docker-compose.redis.yml",
+                "docker-compose.postgres.yml",
```

Comment:
Do we also need the mysql file listed here?

Reply:
no, that is in the invoke.mysql.yml that you can mv to invoke.yml and will take effect.